corehq/apps/user_importer/importer.py

```diff
@@ -532,6 +532,8 @@ def create_or_update_commcare_users_and_groups(upload_domain, user_specs, upload
             if role:
                 role_qualified_id = domain_info.roles_by_name[role]
                 commcare_user_importer.update_role(role_qualified_id)
+            else:
+                commcare_user_importer.update_role('none', clear_role=True)
```

Comment:
I know you didn't make it this way but it's weird to pass in `'none'` as a string

Reply:
It did seem odd. Does it make sense to change it is this PR, to `None`? I think `_AuthoizableMixin` would need to be updated and possibly some methods in other places like. Should that be a followup PR?

Reply:
Not in this PR. Possibly in another PR but only if it's an easy change.